xontrib/vox.py

```diff
@@ -1,77 +1,99 @@
 """Python virtual environment manager for xonsh."""
 
 import sys as _sys
+import argparse as _argparse
 import xontrib.voxapi as _voxapi
 
 class _VoxHandler:
     """Vox is a virtual environment manager for xonsh."""
+    parser = _argparse.ArgumentParser(prog='vox', description=__doc__)
```

Comment:
Ok, wow. I just confirmed this with some quick timeit tests.

```python
timeit('reload(xontrib.vox)', 'from imp import reload; import xontrib.vox', number=10000)
```

With the code as it is (49ea0eee15a1e6cdcf93bdf885560031fca5848d), timeit returns a time of 32.5 seconds. When I comment out parser creation, it takes 1.98s.

(For reference, `timeit('reload(argparse)', 'from imp import reload; import argparse', number=10000)` should be 11.3s, but I had to run it at `number=1000` because 10000 was causing some kind of memory/gc issue.)